### ISSUE TITLE
Adds support for alternate module directory/path

### ISSUE THREAD

**Issue by jantman** [maestrodev/librarian-puppet#15]
We're in the process of transitioning from our legacy puppet modules to new ones. At the moment, we have one git repository with two directories in it - modules/ (our internally developed modules, most of which don't have a Modulefile or anything like that), and modules-community/ which contains our forge and github modules.

It would be really helpful if there was an option to manage a path other than "modules/".

**Comment by carlossg** [comment]
librarian-puppet install --path=something

is that it?

**Comment by jantman** [comment]
Ok, thanks. I was confused by all of the hard-coded "modules/" in the source, but it _seems_ like --path=modules-community/ did what I want. Or at least it installed in the right place, though I'm seeing a lot of changed files even though I created a Puppetfile with the same version specifiers as what I have currently installed. I suppose this is a larger issue with librarian-puppet and a misunderstanding on my part (it's resolving dependencies and installing ones that I had specified at a specific version in my Puppetfile; in one case, both Puppetfile and Puppetfile.lock show "puppetlabs/mcollective (= 1.1.0)" but modules-community/mcollective/Modulefile shows it to be 1.1.3...
